Seneca-PlaNA/body-contouring-clinic
# Eslint prettier config

Fixes: #45 
---
#### How to test this PR?

1. `gh` to your local repo, run `npm install` to install new package for linter and formater.
2. Open a `.js` file and see if your vs code recognize any format error (e.g. if the `import` is using double quotes, you should get an error telling you this should be changed to single quote)
3. There will be many errors from ESLint and Prettier for sure, because we haven't used it in our project. Don't panic if you see a thousand of format errors.

There will be many errors being detected by ESLint and Prettier, which will help us to write better code. Please file issues if you want to change any rules in ESLint or Prettier.

Please test this PR in your local machine and let me know how it goes.
